index.html

```diff
@@ -0,0 +1,154 @@
+    <div class="s1t-main">
+      <div class="s1-space">
+        <div class="s1-img">
+        <img src="like-shirts.svg" alt="웃을 구경하는 판다">
+        </div>
+        <div class="s1-ment">
+          <div class="s1-hotitem">
+          Hot item
+          </div>
+          <div class="s1-best">
+            <div class="s1-bestitem">
+            인기 상품을<br>
+            확인해 보세요
+            </div>
+            <div class="s1-bestcheck">
+            가장 HOT한 중고거래 물품을<br>
+            판다 마켓에서 확인해 보세요
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+<!-- 메인 2번째 -->
+    <div class="s2t-main">
+      <div class="s2-space">
+        <div class="s2-ment">
+          <div class="s2-hotitem">
+            Search
+          </div>
+          <div class="s2-best">
+          <div class="s2-bestitem">
+            구매를 원하는<br>
+            상품을 검색하세요
+          </div>
+          <div class="s2-bestcheck">
+            구매하고 싶은 물품은 검색해서<br>
+            쉽게 찾아보세요
+          </div>
+          </div>
+        </div>
+        <div class="s2-img">
+          <img src="serching.svg">
+        </div>
+      </div>
+    </div>
+<!-- 메인 3번째 -->
+    <div class="s3t-main ">
+      <div class="s3-space">
+        <div class="s3-img">
+          <img src="files.svg">
+        </div>
+        <div class="s3-ment">
+          <div class="s3-hotitem">
+          Register
+          </div>
+          <div class="s3-best">
+          <div class="s3-bestitem">
+          판매를 원하는<br>
+          상품을 등록하세요
+          </div>
+          <div class="s3-bestcheck">
+          어떤 물건이든 판매하고 싶은 상품을<br>
+          쉽게 등록하세요
+          </div>
+          </div>
+        </div>
+      </div>
+    </div>
+<!-- 마지막배너 -->
+    <div class="final-shot ">
+      <div class="fi-mentbox">
+        <div class="fi-ment">
+          믿을 수 있는<br>
+          판다마켓 중고 거래
+        </div>
+      </div>
+      <div class="fi-hipanda">
+        <img src="talking.svg">
+      </div>
+    </div>
+  </main>
+    <!-- 맨 아래 부분 -->
+  <footer class="foot-base ">
+    <div class="foot-line">
+      <div class="foot-box">
+        <div class="foot-ment">
+          ©codeit - 2024 
+        </div>
+      </div>
+      <div class="foot-box2">          
+          <a class="foot-pri" href="/privacy">Privacy Policy</a>                
+          <a  class="foot-faq" href="/faq">FAQ</a>          
+      </div>
+      <div class="foot-box3">
+        <div class="foot-face">
+          <a href="https://www.facebook.com/" target='_blank'><img src="faceb.svg"></a>
+        </div>
+        <div class="foot-twi">
+          <a href="https://x.com/?lang=ko" target='_blank'><img src="twi.svg"></a>
+        </div>
+        <div class="foot-ytub">
+          <a href="https://www.youtube.com/?hl=ko" target='_blank'><img src="ytub.svg"></a>
+        </div>
+        <div class="foot-insta">
+          <a href="https://www.instagram.com/" target='_blank'><img src="insta.svg"></a>
+        </div>
+      </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.
또한 가능하면 classname도 어떤 태그인지 알 수 있는 이름으로 변경하시는 것을 추천드려요~

```suggestion
      <ul class="footer-sns">
        <li class="footer-sns-item">
          <a href="https://www.facebook.com/" target='_blank'><img src="faceb.svg"></a>
        </li>
        <li class="footer-sns-item">
          <a href="https://x.com/?lang=ko" target='_blank'><img src="twi.svg"></a>
        </li>
        <li class="footer-sns-item">
          <a href="https://www.youtube.com/?hl=ko" target='_blank'><img src="ytub.svg"></a>
        </li>
        <li class="footer-sns-item">
          <a href="https://www.instagram.com/" target='_blank'><img src="insta.svg"></a>
        </li>
      </ul>
```